src/components/SpruceForm/index.tsx

```diff
@@ -13,31 +14,37 @@ import widgets from "./Widgets";
 
 export const SpruceForm: React.VFC<SpruceFormProps> = ({
   customFormatFields,
+  customValidate,
   disabled,
   fields,
   formData,
   onChange,
   schema,
   tagName,
   uiSchema,
-  validate,
 }) => (
   <Form
-    fields={{ ...baseFields, ...fields }}
+    fields={fields}
     schema={schema}
     onChange={onChange}
     widgets={widgets}
     uiSchema={uiSchema}
     formData={formData}
     tagName={tagName}
-    ArrayFieldTemplate={ArrayFieldTemplate}
-    FieldTemplate={DefaultFieldTemplate}
-    ObjectFieldTemplate={ObjectFieldTemplate}
+    templates={{
+      ArrayFieldTemplate,
+      FieldTemplate: DefaultFieldTemplate,
+      ObjectFieldTemplate,
+      DescriptionFieldTemplate: baseFields.DescriptionField,
+      TitleFieldTemplate: baseFields.TitleField,
+    }}
     transformErrors={transformErrors}
     showErrorList={false}
-    validate={validate}
+    customValidate={customValidate}
     disabled={disabled}
-    customFormats={customFormats(customFormatFields?.jiraHost)}
+    validator={customizeValidator({
+      customFormats: customFormats(customFormatFields?.jiraHost),
+    })}
     liveValidate
```

Comment:
disabling live validate makes a huge difference in performance so I think the issue might have to do with validation.

Reply:
The docs even indicate that liveValidate has strong performance implications https://rjsf-team.github.io/react-jsonschema-form/docs/usage/validation/#live-validation so it may be worth while to look into the ideal way to do this.

Reply:
Sure, but we have `liveValidate` enabled currently and the performance implications are minor. This is definitely a bug/regression from the library.